apps/useradmin/src/useradmin/views.py

```diff
@@ -73,22 +73,29 @@ def is_ldap_setup():
 
 
 def list_users(request):
-  return render("list_users.mako", request, {
-      'users': User.objects.all(),
+  data = {
       'users_json': json.dumps(list(User.objects.values_list('id', flat=True))),
-      'request': request,
       'is_embeddable': request.GET.get('is_embeddable', False),
       'is_ldap_setup': is_ldap_setup()
-  })
+  }
+
+  # options_json is used in the UI
+  data['options_json'] = json.dumps(data)
+  data['request'] = request
```

Comment:
Sending the whole request back? Is this used?

Reply:
This is similar to existing code, and it is used in the the mako template. Its just that i have removed request from the options_json as it is not required in the UI.